tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/TableName.java

```diff
@@ -26,13 +26,20 @@
     private final Optional<String> schema;
     private final String name;
     private final String nameInDatabase;
+    private final boolean shouldCreateSchema;
 
     public TableName(String database, Optional<String> schema, String name, String nameInDatabase)
     {
+        this(database, schema, name, nameInDatabase, true);
+    }
+
+    public TableName(String database, Optional<String> schema, String name, String nameInDatabase, boolean shouldCreateSchema)
```

Comment:
ahh, that would work. We never want to create schema for TD tables. I'll switch it to that.